bazel/api_build_system.bzl

```diff
@@ -34,6 +38,24 @@ def api_py_proto_library(name, srcs = [], deps = [], has_services = 0):
         visibility = ["//visibility:public"],
     )
 
+def api_go_proto_library(proto, importpath, deps = []):
+    go_proto_library(
+        name = _Suffix(proto, _GO_PROTO_SUFFIX),
+        importpath = importpath,
+        proto = _Suffix(":", proto),
```

Comment:
I'm not sure how much of a win this is, seems clearer to make the `:` explicit to match normal expectation of labels in Bazel.